pcdsdevices/epics_motor.py

```diff
@@ -221,6 +186,85 @@ def check_limit_switches(self):
         else:
             return "Low [] High []"
 
+    def get_low_limit(self):
+        """Get the low limit."""
+        return self.low_limit_travel.get()
+
+    def set_low_limit(self, value):
+        """
+        Set the low limit.
+
+        Parameters
+        ----------
+        value : float
+            Limit of travel in the negative direction.
+
+        Raises
+        ------
+        ValueError
+            When motor in motion or position outside of limit.
+        """
+        if not self.moving:
+            # update EPICS
+            _current_high_limit = self._get_epics_limits()[1]
+            if value <= self.position:
+                if value < _current_high_limit:
+                    self.low_limit_travel.put(value)
+                else:
+                    raise ValueError('Could not set motor low limit to %f. '
+                                     'Low limit must be lower than the '
+                                     'current high limit: %f',
+                                     value, _current_high_limit)
+            else:
+                raise ValueError('Could not set motor low limit to %f at '
+                                 'position %f. Low limit must be lower than '
+                                 'the current position.', value, self.position)
+        else:
+            raise ValueError('Could not set the low limit, the motor is in'
+                             ' motion')
+
+    def get_high_limit(self):
+        """Get high limit."""
+        return self.high_limit_travel.get()
+
+    def set_high_limit(self, value):
+        """
+        Limit of travel in the positive direction.
+
+        Parameters
+        ----------
+        value : float
+            High limit value to be set.
+
+        Raises
+        ------
+        ValueError
+            When motor in motion or position outside of limit.
+        """
+        if not self.moving:
+            # update EPICS
+            _current_low_limit = self._get_epics_limits()[0]
```

Comment:
I'm using the `_get_epics_limits` here, because it will return `0` if not set - is this fine?